packages/itwinui-css/src/backdrop/backdrop.scss

```diff
@@ -24,4 +24,8 @@
   &.iui-backdrop-visible {
     transition-delay: $iui-speed-instant; // remove delay for entry animation
   }
+
+  &.iui-backdrop-contained {
```

Comment:
It would be great to get other native speaker opinion whether this name is ok.

Reply:
i think it should be flipped: `absolute` by default and then this modifier can be called `iui-backdrop-fixed`.

Reply:
I flipped it